mopidy_gmusic/commands.py

```diff
@@ -1,35 +1,34 @@
-from __future__ import print_function
-
-from oauth2client.client import OAuth2WebServerFlow
-
 import gmusicapi
-
 from mopidy import commands
+from oauth2client.client import OAuth2WebServerFlow
 
 
 class GMusicCommand(commands.Command):
     def __init__(self):
-        super(GMusicCommand, self).__init__()
-        self.add_child('login', LoginCommand())
+        super().__init__()
+        self.add_child("login", LoginCommand())
 
 
 class LoginCommand(commands.Command):
     def run(self, args, config):
         oauth_info = gmusicapi.Mobileclient._session_class.oauth
         flow = OAuth2WebServerFlow(**oauth_info._asdict())
         print()
-        print('Go to the following URL to get an initial auth code, then ' +
-              'provide it below: ' + flow.step1_get_authorize_url())
+        print(
+            "Go to the following URL to get an initial auth code, then "
+            + "provide it below: "
+            + flow.step1_get_authorize_url()
+        )
         print()
         try:
-            initial_code = raw_input('code: ')
+            initial_code = raw_input("code: ")
         except NameError:
```

Comment:
If we're moving to only supporting python3, we shouldn't need to fall back here.

Reply:
Good catch. Fixed now!